stl/inc/execution

```diff
@@ -75,42 +75,71 @@ constexpr size_t _Still_active                = static_cast<size_t>(-1);
 
 // EXECUTION POLICIES
 namespace execution {
-    class sequenced_policy { // request for sequential execution with termination
+    class sequenced_policy {
+        // indicates support for only sequential execution, and request termination on exceptions
     public:
         using _Standard_execution_policy   = int;
         static constexpr bool _Parallelize = false;
+        static constexpr bool _Ivdep       = false;
     };
 
     inline constexpr sequenced_policy seq{/* unspecified */};
 
-    class parallel_policy { // request for parallel execution with termination
+    class parallel_policy {
+        // indicates support by element access functions for execution with parallel forward progress guarantees and
+        // requests termination on exceptions
     public:
         using _Standard_execution_policy   = int;
         static constexpr bool _Parallelize = true;
+        static constexpr bool _Ivdep       = true;
     };
 
     inline constexpr parallel_policy par{/* unspecified */};
 
     class parallel_unsequenced_policy {
-        // request for parallel execution without thread identity with termination
+        // indicates support by element access functions for parallel execution with weakly parallel forward progress
+        // guarantees, and requests termination on exceptions
+        //
         // (at this time, equivalent to parallel_policy)
     public:
         using _Standard_execution_policy   = int;
         static constexpr bool _Parallelize = true;
+        static constexpr bool _Ivdep       = true;
     };
 
     inline constexpr parallel_unsequenced_policy par_unseq{/* unspecified */};
+
+#if _HAS_CXX20
+    class unsequenced_policy {
+        // indicates support by element access functions for weakly parallel forward progress guarantees, and for
+        // executing interleaved on the same thread, and  requests termination on exceptions
+        //
+        // (at this time, equivalent to sequenced_policy except for the for_each family)
+    public:

```

Comment:
Pre-existing: Since the classes are unspecified, should we use the same `explicit Meow() = default;` technique that we use in the other tags, to make it slightly harder to unintentionally construct one of these objects?

Reply:
@BillyONeal mentioned that this may affect aggregateness and therefore ABI. As the Standard isn't requiring us to do anything here, and there's no significant user benefit that I can envision, let's disregard this idea.